README.rst

```diff
@@ -326,29 +355,38 @@ For example, the following will build Kokkos for NVIDIA A100 cards
         -DKokkos_ENABLE_EXAMPLES:BOOL=OFF \
         -DKokkos_ENABLE_TESTS:BOOL=OFF \
         -DKokkos_ENABLE_LIBDL:BOOL=OFF
-    cmake --build build && cmake --install build
-    echo export CMAKE_PREFIX_PATH=/opt/kokkos/4.1.00/AMPERE80:\$CMAKE_PREFIX_PATH
+    $ cmake --build build && cmake --install build
+    $ export CMAKE_PREFIX_PATH=/opt/kokkos/4.1.00/AMPERE80:\$CMAKE_PREFIX_PATH
 
 Next, append the install location to ``CMAKE_PREFIX_PATH``.
 Note that the C++20 standard is required (``-DCMAKE_CXX_STANDARD=20`` option), and hence CUDA v12 is required for the CUDA backend.
-If an installation is not found, our builder will clone and install it during the build process.
+
+Install Lightning-Kokkos
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If an installation of Kokkos is not found, then our builder will clone and install it during the build process. Lightning-Qubit should be installed (compilation is not necessary):
 
 The simplest way to install Lightning-Kokkos (OpenMP backend) through ``pip``.
 
 .. code-block:: console
 
-   PL_BACKEND="lightning_kokkos" python scripts/configure_pyproject_toml.py
-   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install .
-
-To build the plugin directly with CMake as above:
+   $ git clone https://github.com/PennyLaneAI/pennylane-lightning.git
+   $ cd pennylane-lightning
+   $ PL_BACKEND="lightning_qubit" python scripts/configure_pyproject_toml.py
+   $ SKIP_COMPILATION=True pip install -e . -vv
+   $ PL_BACKEND="lightning_kokkos" python scripts/configure_pyproject_toml.py
+   $ CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install -e . -vv
 
-.. code-block:: console
+The supported backend options are 
 
-   cmake -B build -DKokkos_ENABLE_OPENMP=ON -DPL_BACKEND=lightning_kokkos -G Ninja
-   cmake --build build
+- ``SERIAL``
+- ``OPENMP``
+- ``THREADS``
+- ``HIP``
+- ``CUDA`` 
```

Comment:
We don't need to review all the backend kernels. We can just refer users to the right Kokkos arch page and it should be enough. What do you think?